crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -150,6 +150,22 @@ def compute_resources_available?
       true
     end
 
+    # Check for presence of HA setup, which is a requirement for non-disruptive upgrade
+    def ha_deployed?
```

Comment:
1. It would be probably better to check more specifically yes (as I've asked in the PR message), but I'm not sure at this point what exactly to check.

2. Yes, error message should be shown, but we're currently redesigning the whole part, so I believe it will come as followup PR that would have unique way of returning errors for all the checks we're currently doing. I believe @MaximilianMeister is already working on something.